### ISSUE TITLE
Fix Topic links

### ISSUE THREAD

**Issue by abdulkdawson** [mitodl/hugo-course-publisher#409]
The Topics are overflowing into the image and it is hard to read as there are no commas separating each topic. 
We should add commas to the end of each topic as well as style the Topics by adding an underline to each one.
Also, can we add 16px of space between the topics section and the course image as diagrammed in the screenshot?
Lastly, can we remove the (s) from the current "Topics" section on the course card and just rename it "Topics".

<img width="751" alt="Screen Shot 2020-11-23 at 3 18 11 PM" src="https://user-images.githubusercontent.com/49685474/100010998-1b89d580-2d9f-11eb-8ce4-1622dcaa10e1.png">

**Comment by pdpinch** [comment]
Can we reduce the number of topics somehow?

**Comment by aliceriot** [comment]
yeah, I'm wondering if we should limit ourselves to a certain number of topics? some courses are tagged with a large number of them.

**Comment by abdulkdawson** [comment]
@aliceriot That would be great. Peter had posed the same question. I guess we should consider a limit.

**Comment by Ferdi** [comment]
@aliceriot , i learned that one of the reason some courses have too many topics is because we merge all topics from all course runs. 
@mbertrand is working on undoing that (each item in the search result will be a course run) . Maybe we can wait after he is done and see how things look.
In any case ,  I don't think we should arbitrary limit the nr of topics in the card.  That would be considered a bug. We can do some expand / collapse thing

**Comment by abdulkdawson** [comment]
@Ferdi ok that makes sense. Thank you for the clarification.
